blueprints/gke/patterns/autopilot-cluster/README.md

```diff
@@ -0,0 +1,211 @@
+- Separate kube-system Pods: automatically managed by Autopilot
+
+### Security
+- Use the security posture dashboard: enabled by default in new clusters
+- Use group authentication: not needed by this blueprint but can be enabled through the `enable_features.groups_for_rbac` variable of the `gke-cluster-autopilot` module.
+- Use RBAC to restrict access to cluster resources: this blueprint deploys the underlying infrastructure, RBAC configuration is out of scope.
+- Enable Shielded GKE Nodes: automatically managed by Autopilot
+- Enable Workload Identity: automatically managed by Autopilot
+- Enable security bulletin notifications: out of scope for this blueprint
+- Use least privilege Google service accounts: this blueprint creates a new service account for the cluster
+- Restrict network access to the control plane and nodes: this blueprint deploys a private cluster
+- Use namespaces to restrict access to cluster resources: this blueprint deploys the underlying infrastructure, namespace handling is left to applications.
+
+### Networking
+- Create a custom mode VPC: this blueprint can optinally deploy a new custom VPC with a single subnet. Otherwise, an existing VPC and subnet can be used.
+- Create a proxy-only subnet: the `vpc_create` variable allows the creation of proxy only subnet, if needed.
+- Configure Shared VPC: by default a new VPC is created within the project, but a Shared VPC can be used when the blueprint handles project creation.
+- Connect the cluster's VPC network to an on-premises network: skipped, out of scope for this blueprint
+- Enable Cloud NAT: the `vpc_create` variable allows the creation of Cloud NAT, if needed.
+- Configure Cloud DNS for GKE: not needed by this blueprint but can be enabled through the `enable_features.dns` variable of the `gke-cluster-autopilot` module.
+- Configure NodeLocal DNSCache: not needed by this blueprint
+- Create firewall rules: only the default rules created by GKE
+
+### Multitenancy
+For simplicity, multi-tenancy is not used in this blueprint.
+
+### Monitoring
+- Configure GKE alert policies: out of scope for this blueprint
+- Enable Google Cloud Managed Service for Prometheus: automatically managed by Autopilot
+- Configure control plane metrics: enabled by default
+- Enable metrics packages: out of scope for this blueprint
+
+### Maintenance
+- Create environments: out of scope for this blueprint
+- Subscribe to Pub/Sub events: out of scope for this blueprint
+- Enroll in release channels: the REGULAR channel is used by default
+- Configure maintenance windows: not configured but can be enabled through the `maintenance_config` in the `gke-cluster-autopilot` module.
+- Set Compute Engine quotas: out of scope for this blueprint
+- Configure cost controls: TBD
+- Configure billing alerts: out of scope for this blueprint
+
+
+<!-- ## Usage Examples -->
+
+<!-- ### Existing cluster with local fleet -->
+
+<!-- ```hcl -->
+<!-- module "jumpstart-0" { -->
```

Comment:
region must specified, shouldn't it? In all the examples

Reply:
Yes, but we hve to re-write all those examples with the new variables. I'll do that in a separate PR